src/Appwrite/Utopia/Response/Model/Func.php

```diff
@@ -49,6 +49,18 @@ public function __construct()
                 'default' => true,
                 'example' => false,
             ])
+            ->addRule('live', [
+                'type' => self::TYPE_BOOLEAN,
+                'description' => 'Is function live (deployed with latest config)?',
```

Comment:
This description is not very clear to me. Live is a new concept we never used before. How is this different from active. The description assumes the reader know a lot about our internal discussions.

Reply:
@Meldiron now that I've reviewed the entire PR, I think a better name would by `sync` or something like that. Live isn't very clear to me and feels like status=active.

Reply:
We stayed with `live` for now, Ill improve docs